benchmarks/test_elemwise.py

```diff
@@ -0,0 +1,55 @@
+import importlib
+import operator
+import os
+
+import sparse
+
+import pytest
+
+import numpy as np
+import scipy.sparse as sps
+
+DENSITY = 0.001
+
+
+def get_test_id(side):
+    return f"{side=}"
+
+
+@pytest.fixture(params=[100, 500, 1000], ids=get_test_id)
+def elemwise_args(request, seed, max_size):
+    side = request.param
+    if side**2 >= max_size:
+        pytest.skip()
+    rng = np.random.default_rng(seed=seed)
+    s1_sps = sps.random(side, side, format="csr", density=DENSITY, random_state=rng) * 10
+    s1_sps.sum_duplicates()
+    s2_sps = sps.random(side, side, format="csr", density=DENSITY, random_state=rng) * 10
+    s2_sps.sum_duplicates()
+    return s1_sps, s2_sps
+
+
+@pytest.mark.parametrize("f", [operator.add, operator.mul, operator.ge])
```

Comment:
This will preserve sparsity in SciPy.

```suggestion
@pytest.mark.parametrize("f", [operator.add, operator.mul, operator.gt])
```

Reply:
Done